.gitignore

```diff
@@ -79,3 +79,5 @@ result
 *.exp
 *.VC.db
 /release
+# Core dumps shouldn't be tracked by git.
+.[09][09]*
```

Comment:
Github says it wants a RET here

Reply:
Yeah, I realized that after I committed it.